nexus_constructor/model/dataset.py

```diff
@@ -37,5 +37,5 @@ def as_dict(self) -> Dict[str, Any]:
             "attributes": [attribute.as_dict() for attribute in self.attributes]
             if self.attributes
             else None,
-            "values": values if values else [],
+            "values": values if values else "",
```

Comment:
I don't think this is the right solution. We'd want "" if it is a string, [] if array and so on. Perhaps this should just be
```
"values": values
```